.travis.yml

```diff
@@ -17,9 +17,8 @@ matrix:
     - env: LANGUAGE=dind-aws     VERSION=1        DOCKER_COMPOSE_VERSION=1.24.0 GLIBC_VERSION=2.27-r0 PIP_VERSION=18.1 PIPENV_VERSION=2018.10.13
     - env: LANGUAGE=golang       VERSION=1.9      GLIDE_VERSION=v0.13.1
     - env: LANGUAGE=java         VERSION=6        MAVEN_VERSION=3.2.1
-    - env: LANGUAGE=java         VERSION=8        JAVA_VERSION=8u212-jdk-slim
-    - env: LANGUAGE=java         VERSION=10       JAVA_VERSION=10.0.2-jdk-slim
-    - env: LANGUAGE=java         VERSION=11       JAVA_VERSION=11.0.3-jdk-slim
+    - env: LANGUAGE=java         VERSION=8        JAVA_VERSION=jdk8u212-b04-debian-slim
```

Comment:
I think we need `JAVA_MAJOR_VERSION` as well?

Reply:
`JAVA_MAJOR_VERSION` is in fact declared in `travis.py` and initlialized from `VERSION` (I didn't add it, it was already done).

Reply:
I'll stop using JAVA_MAJOR_VERSION and use directly VERSION :)

Reply:
`JAVA_MAJOR_VERSION` has been renamed in `VERSION` in `travis.py` & `Dockerfile.tpl`